tests/unit/configFile.spec.js

```diff
@@ -133,4 +133,239 @@ describe("configFile.js", () => {
       expect(result).toStrictEqual(changedSettings);
     });
   });
+
+  describe("parseBasicSettings function", () => {
+    it("data should be parsed correctly", async () => {
+      await LocalStore.clear();
+      const expectedParsedData = {
+        configurationURL:
+          "https://raw.githubusercontent.com/AdvancedThreatAnalytics/threat-analytics-search/main/settings.json",
+        useGroups: true,
+        configEncrypted: false,
+        configEncryptionKey: "",
+        autoUpdateConfig: false,
+      };
+      const result = await ConfigFile.default.parseBasicSettings(
+        defaultSettings.config
+      );
+
+      expect(result).toEqual(expect.objectContaining(expectedParsedData));
+      expect(result.mergeGroups).toEqual(result.useGroups);
+    });
```

Comment:
This test will break if we modify any value on `settings.json`, instead you should do like on the test for `parseGroups`, you define `const rawData = ['someUrl', "true", "false", ...]; const expectedData = { configurationURL: "someUrl", useGroups: true}` and then verify that `parseBasicSettings(rawData) == expectedData`.